src/net.h

```diff
@@ -828,7 +829,7 @@ class CConnman
 
     bool ForNode(NodeId id, std::function<bool(CNode* pnode)> func);
 
-    void PushMessage(CNode* pnode, CSerializedNetMsg&& msg);
+    void PushMessage(CNode* pnode, CSerializedNetMsg&& msg) EXCLUSIVE_LOCKS_REQUIRED(!m_total_bytes_sent_mutex);
```

Comment:
Why not add `AssertLockNotHeld(m_total_bytes_sent_mutex)` in the beginning of `PushMessage()`, `SocketHandler()`, `SocketHandlerConnected()` and `ThreadSocketHandler()`?

Reply:
Good catch. Done. Thanks.